tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -240,7 +244,10 @@ def _process_masks_pytorch(self, boxes, raw_masks, identifiers, original_image_s
         if raw_mask_for_all_classes:
             per_obj_raw_masks = []
             for cls, raw_mask in zip(classes, raw_masks):
-                per_obj_raw_masks.append(raw_mask[cls, ...])
+                if self.scores_out:
+                    per_obj_raw_masks.append(raw_mask[cls, ...])
+                else:
+                    per_obj_raw_masks.append(raw_mask)
```

Comment:
fast fix for linter issue
```suggestion
                    per_obj_raw_masks.append(raw_mask[cls, ...] if self.scores_out else raw_mask)
```